src/Configuration/AccountManager.cs

```diff
@@ -9,42 +10,64 @@ internal static class AccountManager
     {
         public static List<Account> Accounts;
 
-        private static void Load(string serverName)
+        public static string[] GetAccountNames(string serverName)
         {
-            if (Accounts == null)
+            try
             {
-                var accounts = LoadAccountsFromFile();
-                Accounts = accounts.Where(x => x.Server == serverName).ToList();
+                Load(serverName);
+                return Accounts?.Where(x => x.Server == serverName).Select(x => x.UserName).ToArray() ?? new string[] { };
+            }
+            catch(Exception ex)
+            {
+                //Log error?
+                return new string[] { };
             }
-        }
-
-        public static string[] GetAccountNames(string serverName)
-        {
-            Load(serverName);
-            return Accounts?.Where(x => x.Server == serverName).Select(x => x.UserName).ToArray() ?? new string[] { };
         }
 
         public static string GetAccountPassword(string serverName, string userName)
         {
-            Load(serverName);
-            return Accounts?.FirstOrDefault(x => x.UserName == userName)?.Password;
+            try
+            {
+                Load(serverName);
+                return Accounts?.FirstOrDefault(x => x.UserName == userName)?.Password;
+            }
+            catch(Exception ex)
```

Comment:
Hmm why catching Exception here, same above in `GetAccountNames`, I don't think that is necessary, `Load` could gracefully return `bool` or a `Result` object, which then can be handled in these methods. What do you think?

Reply:
yeah i think that would work.  I normally go try/catch heavy where i don't want an unhandled exception to crash the app, but can still produce a default result.  However, you are correct, in this case the try catch can go closer to the file operations.

Reply:
moved the exception handling.